src/mgos_ili9341.c

```diff
@@ -481,6 +483,17 @@ void mgos_ili9341_print(uint16_t x0, uint16_t y0, char *string) {
   free(pixelline);
 }
 
+void mgos_ili9341_printf(uint16_t x0, uint16_t y0, const char *fmt, ...) {
+  char buf[50], *s = buf;
+  va_list ap;
+  va_start(ap, fmt);
+  if (mg_avprintf(&s, sizeof(buf), fmt, ap) > 0) {
```

Comment:
I didn't know this function existed. Very handy! I've come across this "buffer but how large should it be?" problem a number of times, and I'm thrilled to see you've already solved it centrally :)